src/main/kotlin/com/cognifide/gradle/aem/instance/content/ContentManipulator.kt

```diff
@@ -0,0 +1,16 @@
+package com.cognifide.gradle.aem.instance.content
+
+interface ContentManipulator {
+
```

Comment:
how about adding 

```kotlin
fun setProperty(path: String, name: String, value: Any?)
fun removeProperty(path: String, name: String) = setProperty(path, name, null)
fun getProperty(path: String, name: String): Any
fun findProperty(path: String, name: String): Any?
fun hasProperty(path: String, name: String) = findProperty(path, name) != null
```

I think that  custom tasks will most often only update single properties so that dedicated methods would be useful. e.g to disable some workflow then `setProperty("/etc/workflow/launchers/dam_update_asset", "enabled", false)` will be called